packages/sanes-chrome-extension/src/context/PersonaProvider.tsx

```diff
@@ -44,5 +47,25 @@ export const PersonaProvider = ({ children }: Props): JSX.Element => {
     update: loadPersonaInReact,
   };
 
+  if (isExtensionContext) {
+    chrome.runtime.onMessage.addListener((message, sender, _sendResponse) => {
```

Comment:
I believe this is the only penalty we have to pay because we do not use redux.

Do you see another cleaner mechanism to update txs besides this back and forth of messaged between BS - persona - PersonaProvider?

Reply:
Passing messages is the only possible communication method between BS and UI, which are two independent JavaScript processes.

redux can possibly wrap this my hiding the messaging, but I have no idea if and how that would work.

Reply:
At the end we are coding a chrome extension, so after thinking a bit, I think it is ok to have this listener in a react component. We have a great benefit of not using redux much less boilerplate code to maintain.

Once we have finished with everything we can think about other solutions.

One comment, seeing you have placed this in a react component, can you try to wrap this method in a React.useEffect? I do not want to subscribe more than one listener for this in case the component is re-rendered. Also, can you unsubscribe returning a function? Something like:
```
React.useEffect(() => { 
  if (isExtensionContext) {
    chrome.runtime.onMess....
    return chrome.runtime.onMessage.removeListener() <-- not sure if this does exist or is necessary.
 }
```